frontend/src/components/Button.js

```diff
@@ -1,17 +1,17 @@
 import * as React from "react";
 import { Link } from "gatsby";
 import { ButtonArrow } from "./svgs";
-import { button, textContainer } from "../css/button.module.css";
+import { button, textContainer, buttonSecondary } from "../css/button.module.css";
 
-export const ButtonLink = ({ to = "#", hasArrow = false, children }) => (
-  <Link className={button} to={to}>
+export const ButtonLink = ({ to = "#", hasArrow = false, children, primary = true }) => (
```

Comment:
I feel like it's more clear either to make a component specifically called `PrimaryButtonLink` or `SecondaryButtonLink` (as I did in my PR), or to pass in the string `'primary'` or `'secondary'` explicitly to specify the style, instead of having to know that `primary = false` makes the button look a certain way. But it's not a big deal, whatever works

Reply:
I agree with what @jchenggggg  has because we might have more customization of buttonlink in the future.  once @jchenggggg merge his into main you can rebase and use his logic.

Reply:
Will do!

Reply:
I merged #28 !